bucket/tuxguitar.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.5.3",
+    "description": "TuxGuitar is a multitrack guitar tablature editor and player written in Java-SWT, It can open GuitarPro, PowerTab and TablEdit files.",
+    "homepage": "http://www.tuxguitar.com.ar/",
+    "license": "GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later.html)",
+    "suggest": {
+        "JRE": "java/adopt8-hotspot-jre"
+    },
+    "architecture": {
+        "32bit": {
+            "url": "https://downloads.sourceforge.net/projects/tuxguitar/files/TuxGuitar/TuxGuitar-1.5.3/tuxguitar-1.5.3-windows-x86.zip",
+            "hash": "sha1:390356aa044dbe332ce289b8461aa50e7afdf730",
+            "extract_dir": "tuxguitar-1.5.3-windows-x86"
+        }
+    },

```

Comment:
```suggestion
"url": "https://downloads.sourceforge.net/projects/tuxguitar/files/TuxGuitar/TuxGuitar-1.5.3/tuxguitar-1.5.3-windows-x86.zip",
"hash": "sha1:390356aa044dbe332ce289b8461aa50e7afdf730",
"extract_dir": "tuxguitar-1.5.3-windows-x86"
```

Reply:
I considered to put it inside the 32bit tag because the program only works with this version of JRE.

Reply:
@ferxohn it does not support 64 bit JRE?

Reply:
No. At least this version (tuxguitar 1.5.4) only supports JRE 32 bits.

Reply:
v1.5.4 only supports 32bit JVM, too.